scripts/update-docker.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Used to restart a single msgr api container running on a server
+# First kills all running containers named msgr_api then starts a new container with the name msgr_api
+
+msgr_containers=$(docker ps -q --filter name="msgr_api")
+
+if [ ! -z $msgr_containers ] 
+then
+    echo "Killing all running docker processes"
```

Comment:
Could you update this output since the script is just killing the one container?